src/modules/tests/test-summary/cat-cpc/test-summary.cat-cpc.reducer.ts

```diff
@@ -0,0 +1,43 @@
+import { createFeatureSelector } from '@ngrx/store';
+import * as fromCPCTestSummaryActions from './test-summary.cat-cpc.actions';
+import * as fromTestSummaryActions from '../common/test-summary.actions';
+import { TestSummary } from '@dvsa/mes-test-schema/categories/CPC';
+
+export const initialState : TestSummary = {
```

Comment:
initialState will need to have D255: false

Reply:
added